src/rust-hello/src/lib.rs

```diff
@@ -0,0 +1,17 @@
+#![no_std]
+
+use core::panic::PanicInfo;
+use libc::{abort, puts};
```

Comment:
What's the point of using Rust if you have to use libc?

Reply:
It's what we have to do in order to statically link Stable Rust into C. When that changes, so shall this.

It's not all bad, though. There are still crates we can pull in to make things nicer.

Reply:
If we can't use Rust's standard library, then what's the point of using Rust?

Reply:
Macros, better types, memory safety, `async`/`await`, it looks nicer, ...